### PR TITLE
[Config] Rename `evaluation_frequency` to `eval_frequency`

### PR DESCRIPTION
**Pull request by isratnisa**
Resolves https://github.com/awslabs/graphstorm/issues/143

### CODE REVIEW

File: python/graphstorm/config/argument.py [hunk 1]
```diff
@@ -1450,11 +1450,11 @@ def _add_hyperparam_args(parser):
     # control evaluation
     group.add_argument("--eval-batch-size", type=int, default=argparse.SUPPRESS,
             help="Mini-batch size for computing GNN embeddings in evaluation.")
-    group.add_argument('--evaluation-frequency',
+    group.add_argument('--eval-frequency',
             type=int,
             default=argparse.SUPPRESS,
             help="How offen to run the evaluation. "
-                 "Every #evaluation_frequency iterations.")
+                 "Every #eval_frequency iterations.")
```
Comment (zhjwy9343) [inline python/graphstorm/config/argument.py:1457]: Can we change the "eval_frequency" to "eval-frequency"?

File: python/graphstorm/model/node_gnn.py [hunk 2]
```diff
@@ -218,15 +219,19 @@ def node_mini_batch_predict(model, emb, loader, return_label=False):
             assert len(input_nodes) == 1, "Currently we only support one node type"
             ntype = list(input_nodes.keys())[0]
             in_nodes = input_nodes[ntype]
-            pred = model.decoder.predict(emb[ntype][in_nodes].to(device))
+            print("change here")
```
Comment (zhjwy9343) [inline python/graphstorm/model/node_gnn.py:222]: Are the changes in this node_gnn.py related to the evaluation_frequency argument change?

If not, please separate this change from the current PR.

Thanks.

**Review by zhjwy9343 — CHANGES_REQUESTED** [review]

File: python/graphstorm/model/node_gnn.py (hunk 2 above)
Comment (isratnisa) [reply python/graphstorm/model/node_gnn.py:222]: My bad. This file is modified by mistake.

File: python/graphstorm/config/argument.py (hunk 1 above)
Comment (isratnisa) [reply python/graphstorm/config/argument.py]: Fixed.

**Review by zhjwy9343 — APPROVED** [review]
LGTM

**Pull request merged**